cmd/grpc/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	grpc2 "github.com/stasBigunenko/monorepa/pkg/grpc"
+	pb "github.com/stasBigunenko/monorepa/pkg/grpc/proto"
+
+	"github.com/stasBigunenko/monorepa/pkg/storage"
+	"google.golang.org/grpc"
+	"net"
+	"os/signal"
+	"syscall"
+)
+
+type Config struct {
+	gRPCServAddress string
+}
+
+func getConfig() Config {
+	grpcServAddr := os.Getenv("GRPC_SERV_ADDRESS")
+	if grpcServAddr == "" {
+		grpcServAddr = "127.0.0.1:50052"
+	}
+
+	return Config{
+		gRPCServAddress: grpcServAddr,
+	}
+}
+
+func main() {
+	config := getConfig()
+
+	lis, err := net.Listen("tcp", config.gRPCServAddress)
+	if err != nil {
+		log.Fatalf("failed to listen: %s", err)
+	}
+
+	d := storage.NewStorage()
+	data := storage.ItemService(d)
+
+	// create GRPC server
+	s := grpc.NewServer()
```

Comment:
You can init this part at transport layer. It will exclude grpc/grpc2 names in same package.
Can init this at pkg/grpc/server.go file(now it is empty)

Reply:
I think it's okay to initialize basic variables in the main file. It is rather a bug of naming. The package should be called grpcservice, and this will resolve the "grpc2" bug